src/migration/PoC_dfusion.js

```diff
@@ -10,49 +22,32 @@ async function migrate({ artifacts, deployer, network, account, web3, maxTokens
       account,
       web3,
     })
+
+    const TokenOWLProxy = artifacts.require("TokenOWLProxy")
+    feeToken = await TokenOWLProxy.deployed()
```

Comment:
`feeToken = await deployOwl(...)` should work by itself, right?

Reply:
I checked it. no it does not work.
```
TypeError: Cannot read property 'address' of undefined
    at migrate (/Users/alexherrmann/gnosis/dex-contracts/src/migration/PoC_dfusion.js:47:60)
    at process._tickCallback (internal/process/next_tick.js:68:7)
Truffle v5.1.10 (core: 5.1.10)
```
address of feeToken can not be found